gpt_computer_agent/api.py

```
@@ -633,6 +728,76 @@
+@app.route("/stop_server", methods=["POST"])
+def stop_server():
+
+
+    try:
+        try:
+            from .gpt_computer_agent import the_main_window
+            the_main_window.close()
+        except ImportError:
+            from gpt_computer_agent import the_main_window
+            the_main_window.close()
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)